.github/workflows/build-deploy-api-v2.yaml

```diff
@@ -161,3 +161,21 @@ jobs:
           NAMESPACE: "${{ secrets.OPENSHIFT_DEV_NAMESPACE }}"
         run: |
           ./.github/helpers/cleanup-rc.sh
+
+  DEV-Success-Notify:
+    runs-on: unbuntu-latest
+    environment: API-Dev
+    needs: [ DEV-cleanup ]
+    steps:
+
+        - name: Message Teams Channel on Success
+          run: |
+            curl --location --request POST ${{ secrets.TEAMS_SUCCESS_URL }} \
+            --header 'Content-Type: text/plain' \
+            --data-raw '{
+              "@context": "https://schema.org/extensions",
+              "@type": "MessageCard",
+              "themeColor": "0072C6",
+              "title": "SUCCESS! Deployment to PIA ${{ secrets.ENVIRONMENT }} Environment has finished.",
+              "text": "The PIA API has been built and deployed to the ${{ secrets.ENVIRONMENT }} environment. \n\n The logs for the successful deployment can be found (here)[${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}]\n\n The changes for commit ${{ github.sha }} can be found [here](https://dev.pia.gov.bc.ca/api/api-docs).",
```

Comment:
should this be content? 
https://learn.microsoft.com/en-us/graph/api/channel-post-messages?view=graph-rest-1.0&tabs=http

Reply:
I'm working of the example found here:

https://learn.microsoft.com/en-us/outlook/actionable-messages/send-via-connectors

Reply:
![image](https://user-images.githubusercontent.com/8980101/215860369-20ee1bef-95c0-4a21-b035-ac9e9c4954bd.png)

I think some of the values are not de-referenced.